### PR TITLE
Create FriendsListTentative

### PR DESCRIPTION
**Pull request by Acquiescence3**
I found this coding for a friends list on a side server's GitHub (coding is by Volco). Maybe this could be adapted to work on main?

### CODE REVIEW

**Comment by Volco** [comment]
Yo, don't steal people's code and try and pass it as your own

**Comment by Acquiescence3** [comment]
I'm sorry. I honestly was not trying to pass this code off as my own. I nust thought it would possibly serve as a basis for a friends list on the main server.

**Comment by Volco** [comment]
Friends list for main would be a clientside feature most likely so this would not be good enough for a baseline. Also you 100% were trying to pass it off as yours until you editted the initial comment after i called you out on it. Can someone with the ability to close PRs close this? This isn't even how main wants the friends list done in the first place making this a useless PR

**Comment by Kaiepi** [comment]
For reference for people looking at this pullreq, friends lists are already part implemented on the client. Any code dealing with them on the server would need to make requests to the client, not store them locally.

**Comment by Acquiescence3** [comment]
Sorry about the trouble here. I should have thought this through more before hastily opening a request here. I will work on improving my coding skills before submitting any more requests here.

File: FriendsListTentative [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/****************************
+ * Friend List
+ * Original idea from WGC
+ * Coded by Volco
```
Comment (AustinXII) [inline FriendsListTentative]: On the topic of “do not try and pass other people’s code as your own.” Is this yours? I swear this is Phil’s or JD’s code, no?

Comment (Acquiescence3) [reply FriendsListTentative]: It is not mine. For some dumb reason I thought it wouldn't matter whether I included the author of the code or not. I definitely know better now.

Comment (Volco) [reply FriendsListTentative]: No austin, i made this for WL. Jd's is probably still on wisps repo if he has it still and i don't even remember when phils is.
Compare it if you want but i did make it from scratch

**Comment by Slayer95** [comment]
@Zarel , this is the third unethical PR from @Acquiescence3 . I propose blocking.

**Comment by scheibo** [comment]
I proposed that as well on Discord. I don't think we want Kirkburn as a contributor

**Comment by Zarel** [comment]
There's a sense in which I'm more open to storing friends lists on the individual servers.

And this time it didn't _look_ like he was intentionally trying to pass it off as his own.

But none of this is _useful_ and I see no reason not to block him.

**Comment by Volco** [comment]
He only made it look like he wasn't when i called him out on it before the initial PR did say "he made the code" at the time of opening

**Comment by Zarel** [comment]
Before editing, it said: "I found this coding for a friends list on a side server's GitHub."

"Found", not "made". The "Coded by Volco" part also seems to have been part of the comment from the beginning.

(I blocked him anyway because this is entirely the wrong place for the file, and he doesn't seem to know anything about programming at all that would make his contributions useful.)

File: FriendsListTentative (hunk 1 above)
Comment (AustinXII) [reply FriendsListTentative]: Fair enough, it does seem to be heavily based off jd’s but you’ve done enough to warrant it to be your own

Comment (Volco) [reply FriendsListTentative]: I didn't even look at jd's at all tho lol, but w/e

**Pull request closed without merging**